nix/bootstrap.sh

```diff
@@ -27,7 +27,21 @@ for i in "$BASEDIR"/*; do
         FILE="$(basename "$j")"
         BASEFILE="$HOME/.$FILE"
  
-        if [ -f "$BASEFILE" -o -h "$BASEFILE" ]; then
+        if [ -h "$BASEFILE" ]; then
+           echo "Updating link : $BASEFILE"
+           rm "$BASEFILE"
+        elif [ -d "$BASEFILE" ]; then
+           echo "Replacing directory: $BASEFILE (saving old version)"
+           SAVE_NAME="$BASEFILE.dotfiles.sav"
+           if [ -e "$SAVE_NAME" ]; then
+              SAVE_NAME="$SAVE_NAME.$(date +'%s')"
+           fi
+           mv "$BASEFILE" "$SAVE_NAME"
```

Comment:
I'm just not sure the directory aspect is necessary. For you to hit this, you would need to:

1. Set up some file in your dotfiles settings, AND
1. Have a directory of the identical name

Does this come up a lot? I've never had this happen to me ...